### PR TITLE
ci-operator: Update OLM 4.11+ e2e related jobs to be agnostic of cloud platform

### PR DESCRIPTION
**Pull request by timflannagan**

### CODE REVIEW

File: ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml [hunk 1]
```diff
@@ -74,9 +74,9 @@ tests:
   commands: make unit/api
   container:
     from: ci-image
-- as: e2e-aws-olm
+- as: e2e-agnostic-olm
```
[the remark this answers is not in the document]
Comment (bparees) [reply ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml:77]: it's purely a convention.  The jobs themselves get manually rebalanced across clouds by the DPTP team, as i understand it.  This is just a way to tell dptp "if you need to move this job to another cloud, it's fine, it doesn't matter which platform it runs on"

[the remark this answers is not in the document]
Comment (timflannagan) [reply ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml:77]: Yep - what's strange is the referenced PR for the CVO swaps e2e-gcp -> e2e, but if you look at their prowjob config (and monitorings) they both have "agnostic" in their operator e2e names. So maybe that's just out-of-date, or more realistically it's what Ben mentioned above where this is just a convention right now.

[the remark this answers is not in the document]
Comment (timflannagan) [reply ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml:77]: FYI - I ended up splitting the move from aws -> gcp into it's own separate PR: https://github.com/openshift/release/pull/27253.

[the remark this answers is not in the document]
Comment (petr-muller) [reply ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml:77]: @bparees Nopwe

**Pull request closed without merging**